### PR TITLE
Remaps TCFL ship + adds new docking arm

### PR DESCRIPTION
**Pull request by kyres1**
Remaps the TCFL ship entirely. Overhauls a lot of its former sections and redoes the shuttle entirely. Also, adds one more spawn point for legionnaires, while removing the prisoner roles entirely. However, the brig is still there.

This affects the longevity of the shuttle and ship substantially by giving them static atmos containers of carbon dioxide, as well as spares in case an extra injection is needed. I'm willing to adjust these, though I'm not entirely sure why the stockpile was so low to begin with.

Plus, adds a new "docking arm" (seen below) where shuttles that are different shapes than a literal 5x9 rectangle can dock. This is to circumvent the hangars, though I'm sure smaller shuttles can easily still exist. 

docking arm

To-do : 

- [x] Decimate the prisoner roles.

- [x] changelog........................

shipcomplete

### CODE REVIEW

**Comment by kyres1** [comment]
!review

**Comment by GrayRachnid** [comment]
How does their new shuttle dock with the horizon? It looks too big for the hanger bays.

**Comment by kyres1** [comment]
> How does their new shuttle dock with the horizon? It looks too big for the hanger bays.
> 
> Edit: nevermind I see you added a thing on the horizon just for them. What's wrong with using the pre-existing docking arms? Edit2: You don't need 3 air cans on that docking arm. Two or 1 will suffice.

The pre-existing docking arms are occupied by much more frequent visitors (mercs, raiders etc.) and have awkward angles or are shoved up against a wall, pretty much.

**Comment by GrayRachnid** [comment]
Deck 2
image

Deck 3
image

I feel like all 3 of these can either be widened by 1 tile or the TCFL can be doors can be made smaller by one. Keep in mind, the doors would be wide open on docking so people can just funnel through easily. It'd just be a 1x6 hall instead of 2x6.

I don't think Merc/Burglar docking should be a concern because there's like a total of 9 different entrances, including ones from space. Not to mention their shuttles can appear anywhere around the Horizon while TCFL is forced to dock somewhere. Adding a whole arm for this change feels excessive to me when there's 3 south-facing ones that can already accommodate them.

Also. Don't Merc/Burglar shuttles have their doors on the east? They wouldn't conflict.

**Review by Ben10083 — COMMENTED** [review]
image
two mech rechargers but no mechs, it doesn't make too much sense to have two rechargers even if they had a mech however, perhaps replace one of them with something else like a cyborg recharger?

**Review by Ben10083 — CHANGES_REQUESTED** [review]
image
These cyropods dont have a console, they won't put them into storage. Orion Express ship has consoles and you can see how they were used so you can add them here

**Review by Ben10083 — COMMENTED** [review]
image
In case of a atmos breach at the airlock , this would vent a decent chunk of the ship, perhaps add a double glass airlock at the two tile wide entrance?

**Comment by Ben10083** [comment]
image
What is this room out of curiosity? what are those consoles for.

**Review by Ben10083 — CHANGES_REQUESTED** [review]
image
I see what is the knife and what I am assuming the webbing, but hard for me to tell.

Prefect and legionnaires use pouches not webbing currently (however if you want to change that if people want that, thats whatever,) and if you want the Prefect's items to be given to them on the rack, you will have to update their backpack contents in the ghostroles.dm

**Comment by TheDanseMacabre** [comment]
> image I see what is the knife and what I am assuming the webbing, but hard for me to tell.
> 
> Prefect and legionnaires use pouches not webbing currently (however if you want to change that if people want that, thats whatever,) and if you want the Prefect's items to be given to them on the rack, you will have to update their backpack contents in the ghostroles.dm

This is an energy shield.

**Comment by kyres1** [comment]
Shuttle changed. The docking arm no longer exists, the nav point was moved to deck 3 and now lands on top of the Horizon. Ben's changes are addressed as well, with exception of giving more guns to the TCFL.

**Review by Ben10083 — APPROVED** [review]

File: html/changelogs/kyres1-#14087.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - maptweak: "Remapped the TCFL Peacekeeper ship, and its respective shuttle."
+  - maptweak: "Adds a new docking arm outside the Operations Lobby where big shuttles of different shapes can dock."
```
Comment (Ben10083) [inline html/changelogs/kyres1-#14087.yml:42]: This has changed now right?

Comment (Ben10083) [reply html/changelogs/kyres1-#14087.yml:42]: the docking arm, I mean.

**Comment by WickedCybs** [comment]
> After reviewing the map the docking arm here image
> 
> The one on the left, is unoccupied by any ship. I think you can either modify it or the TCFL ship. Hell, you can probably leave them both as they are and it'd still function.

The dock to the left is used by the merchant if it spawns.

**Comment by Arrow768** [comment]
I am not sure if adding yet another docking location is such a good idea.
Imho it would be better to expand the security checkpoint a bit, so the TCFL also docks "behind" the security checkpoint.

Long term we should reconsider how we handle docking with the horizon.

**Comment by kyres1** [comment]
ILoveDocking

Docking location adjusted to be here, rather than under cargo

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**